iron-overlay-manager.html

```diff
@@ -45,7 +45,11 @@
     }
 
     function currentOverlay() {
-      return overlays[overlays.length-1];
+      var i = overlays.length - 1;
```

Comment:
looks like starting i at the last index in the overlays array and walking backwards through it until it finds one that is opened. :goat: :palm_tree: :maple_leaf: